packages/dataviews/src/components/dataviews-filters/search-widget.tsx

```diff
@@ -108,18 +131,24 @@ function ListBox( { view, filter, onChangeView }: SearchWidgetProps ) {
 				filter.name
 			) }
 			onFocusVisible={ () => {
-				if ( ! compositeStore.getState().activeId ) {
-					compositeStore.move( compositeStore.first() );
+				// `onFocusVisible` needs the `Composite` component to be focusable,
+				// which is implicitly achieved via the `virtualFocus: true` option
+				// in the `useCompositeStore` hook.
```

Comment:
Still valuable. It explains why `onFocusVisible` is actually working, because from the prop docs one may assume it won't fire since the `focusable` prop is not set